README.md

```diff
@@ -1,95 +1,109 @@
-# Operate First - Site Reliability Engineering
+# SIG-SRE — the Operate First Site Reliability Engineering Special Interest Group
 
 ## Introduction
 
-This the Operate First repository for Site Reliability Engineering (SRE)
-documentation. Here we intend to collect documentation on the nitty-gritty
-aspects of building and operating an effective SRE organisation.
+When starting the Operate First project, we looked around and said, "There is no Open group of SRE practitioners.
+Nowhere to discuss and document all the Why, What, and How of our discipline.
+We should fix that."
 
-## Who runs this repository?
+So we are fixing that, and you are invited to participate in creating this [community of practice](https://en.wikipedia.org/wiki/Community_of_practice) around Site Reliability Engineering (SRE).
 
-Hello! We're the Red Hat SIG-SRE group, a cross-team group of people inside Red
-Hat with an interest in promoting and developing strong SRE culture. As a
-result you can expect some of the material here to be a little slanted toward
-the way we do things here inside Red Hat, but as we're doing our best to stick
-to what are generally accepted to be good SRE practices you should be able to
-make use of it with, at most, minor modification.
+This is SIG SRE, the Special Interest Group (SIG) for Site Reliability Engineering (SRE) in the Operate First project.
+
+We focus on a core part of the Operate First goal to improve managed services by fully Opening all aspects of the cloud environment.
```

Comment:
With a particular focus on observability, managing fleets, incident response, operability and similar core disciplines.